app/views/mountain_view/styleguide/show.html.erb

```diff
@@ -2,20 +2,33 @@
   <h1><%= @component.title %></h1>
 </div>
 
-<% @component.styleguide_stubs.each_with_index do |component_properties, index| %>
-  <div class="mv-component">
-    <div class="mv-component__item">
-      <%= render_component @component.name, component_properties %>
-    </div>
-    <div class="mv-component__description">
-      <h2><%= @component.title %> <%= index + 1 %></h2>
-      <div class="mv-component__description__definition">
-        <code class="language-ruby">&lt;%= render_component("<%= @component.name %>", {properties as below}) %&gt;</code>
+<% if @component.styleguide_stubs %>
```

Comment:
What do you think of adding a `stubs?` method for this.

There could be a case where `styleguide_stubs` returns an empty array, and that would evaluate to true but not show any examples. We can make more complex checks in a new method, without polluting the view.